layouts/DocsLayout.tsx

```diff
@@ -1,20 +1,20 @@
 import BaseLayout from './BaseLayout';
 import SideNavigation from '../components/SideNavigation';
-import { useNodeData } from '../hooks/useNodeData';
+import { useNodeReleasesData } from '../hooks/useNodeReleasesData';
 import type { FC, PropsWithChildren } from 'react';
 
 const DocsLayout: FC<PropsWithChildren> = ({ children }) => {
-  const { currentLtsVersion, currentNodeVersion } = useNodeData();
+  const { lts, current } = useNodeReleasesData();
```

Comment:
Using nullable checks here isn't the best idea, because otherwise the versions are `vundefined.x`

You should probably have fallbacks...

```tsx
  const translationContext = {
    apiLts: {
      ltsNodeVersion: lts ? `v${lts.major}.x` : undefined,
      fullLtsNodeVersion: lts ? `v${lts.version}` : undefined,
      spanLts: <span className="small color-lightgray">LTS</span>,
    },
    apiCurrent: {
      fullCurrentNodeVersion: current `? `v${current.version}` : undefined,
      currentNodeVersion: current ? `v${current.major}.x` : undefined,
    },
  };
```